middle_end/flambda2/simplify_shared/call_site_inlining_decision_type.ml

```diff
@@ -164,6 +169,13 @@ let report_reason fmt t =
     Format.fprintf fmt "the@ call@ has@ an@ attribute@ forbidding@ inlining"
   | Attribute_always ->
     Format.fprintf fmt "the@ call@ has@ an@ [@@inline always]@ attribute"
+  | Replay_history_says_must_inline ->
+    (* CR gbury: We should probably not include in the inlining report inlining
+       decisions that were made during replays (e.G. continuation
+       specialization). *)
```

Comment:
If we keep the inlining decisions themselves in the replay history, we wouldn't need a special reason here. That could remain for later, however.

Reply:
Comment updated to reflect that possibility.